test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1954,14 +1954,17 @@ def test_quantized_linear_with_pointwise_binary(
             return
         B = (2, batch_size) if input_3d else (batch_size,)
         input = torch.randn(*B, in_features).to(dtype=torch.float32)
+        input2 = torch.randn(*B, in_features).to(dtype=torch.float32)
+        input3 = torch.randn(*B, out_features).to(dtype=torch.float32)
 
         other = torch.randn(*B, out_features).to(dtype=dtype)
-        # Avoid hitting qlinear inplace sum fusion
```

Comment:
Looks like this comment explains how `other2` is created and reshaped to avoid inplace sum pattern. And it still holds here. Inplace sum is handled in the new `M2`. So, maybe we should keep it? Same below.